src/platyPS/platyPS.psm1

```diff
@@ -1512,7 +1512,20 @@ function GetMarkdownFilesFromPath
         }
     }
 
-    return $MarkdownFiles
```

Comment:
If we are going this route of the more intelligent heuristic, we should remove the old dash-based heuristic.